rdf/rdf12/rdf-semantics/test000s.ttl

```diff
@@ -1,3 +1,3 @@
 prefix : <http://example.com/ns#>
 
-<< :a :b :c >> :p1 :o1.
+<<( :a :b :c )>> :p1 :o1.
```

Comment:
It appears this should be removed (or moved to turtle and marked as negative/syntax error?).

Reply:
This wasn't listed in entries, and there seem to be other which are defined and not listed. I'll remove this, and add the missing entries into `mf:entries`.

Reply:
There were a number of tests which were not referenced in entries and the comment said that they were withdrawn as the current semantics didn't support the test. I removed the entries and test files.